docs/installation.rst

```diff
@@ -39,8 +39,14 @@ environments should consider using other packages that help manage them such as
 `pyenv-virtualenvwrapper <https://github.com/yyuu/pyenv-virtualenvwrapper>`_.
 
 
+Sources
+-------
+
+Girder can be installed from two sources, **pip** or **git repository**.
```

Comment:
En, I personally don't know how to install from git with pip, I always have been cloning with git. 
Would you share some details on this? or make the change directly if you prefer.